utils/check_template_test_coverage.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+from glob import glob
+import os.path
+import re
+import sys
+
+import iniparse
+
+# This script checks that all starting conditions of all tribes and all
+# win conditions are used in the regression test suite.
+
+print('Checking starting and win conditions test coverage')
+
+basedir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir))
+
+datadir = os.path.join(basedir, 'data')
+if not os.path.isdir(datadir):
+    print('ERROR: Data directory ' + datadir + ' not found.')
+    sys.exit(1)
+
+# Collect win conditions
+wcdir = os.path.join(datadir, 'scripting', 'win_conditions')
+wc_init = os.path.join(wcdir, 'init.lua')
+if not os.path.isfile(wc_init):
+    print('ERROR: Win conditions config file not found at ' + wc_init)
+    sys.exit(1)
+
+with open(wc_init, 'r', encoding = 'utf-8') as fh:
+    wc_regexp = re.compile('("[a-z_]+\\.lua")')
+    win_conditions = {match.group(1): False
+                      for match in [wc_regexp.search(line)
+                      for line in fh.read().splitlines()] if match}
+if len(win_conditions) < 1:
+    print('ERROR: No win conditions found.')
+    sys.exit(1)
+
+print()
+print('Win conditions:')
+for wc in win_conditions:
+    print('   ' + wc)
+
+# Collect start conditions for all tribes
+tribesdir = os.path.join(datadir, 'tribes', 'initialization')
+tribes = sorted([td for td in glob(os.path.join(tribesdir, '*'))
+                if os.path.isdir(os.path.join(td, 'starting_conditions'))])
+if len(tribes) < 1:
+    print('ERROR: No tribe directories found.')
+    sys.exit(1)
+
+start_conditions = {'"' + os.path.basename(td) + '"':
+                    {'"' + os.path.basename(sc) + '"': False
+                     for sc in sorted(glob(os.path.join(td, 'starting_conditions', '*')))}
+                    for td in tribes}
+
+if len(start_conditions) < 1:
+    print('ERROR: No start conditions found.')
+    sys.exit(1)
+
+print()
+print('Start conditions:')
+for tr in start_conditions:
+    print('   ' + tr + ':')
+    for sc in start_conditions[tr]:
+        print('      ' + sc)
+
+# Collect tests
+testdir = os.path.join(basedir, 'test', 'templates')
+tests = sorted(glob(os.path.join(testdir, '*.wgt')))
+if len(tests) < 1:
+    print('ERROR: No tests found in directory ' + testdir)
+    sys.exit(1)
+
+unknown_wc = []
+unknown_tribe = []
+unknown_sc = dict()
+
+# Collect uses
+for test in tests:
+    with open(test, 'r', encoding = 'utf-8') as fh:
+        cfg = iniparse.INIConfig(fh)
+
+    for s in cfg:
+        section = cfg[s]
+
+        # Test's win condition
+        if s == 'global':
+            if not section.win_condition in win_conditions:
```

Comment:
The only change for `configparser` is that all instances of `section.xxx` must be changed to `section["xxx"]`. Should not harm much.